GramAddict/__init__.py

```diff
@@ -161,6 +161,11 @@ def run():
         session_state.args = args.__dict__
         sessions.append(session_state)
 
+        attempts = 0
+        while not device.is_alive() and attempts < 5:
+            DeviceFacade(device_id).get_info()
```

Comment:
it fixes the connection on first attempt - pausing until the connection recovers, but give it a few extras just in case.

Reply:
This could in theory just be changed to `device_info = DeviceFacade(device_id).get_info()`, but there is no failsafe then if we need to try again. The while checks agents health.